doc/source/whatsnew/v1.0.0.rst

```diff
@@ -118,7 +118,7 @@ Datetimelike
 - Bug in :meth:`pandas.core.groupby.SeriesGroupBy.nunique` where ``NaT`` values were interfering with the count of unique values (:issue:`27951`)
 - Bug in :class:`Timestamp` subtraction when subtracting a :class:`Timestamp` from a ``np.datetime64`` object incorrectly raising ``TypeError`` (:issue:`28286`)
 - Addition and subtraction of integer or integer-dtype arrays with :class:`Timestamp` will now raise ``NullFrequencyError`` instead of ``ValueError`` (:issue:`28268`)
--
+- New test to ensure OutOfBoundsDatetime exception is raised when calling :func:``pd.to_datetime`` with out-of-bounds date strings
```

Comment:
this note is not necessary since the change is not user-facing

Reply:
Though no real harm in including it. Your call @squigmig.

If you include it, you'll need to refer to an issue with :issue: and fix the formatting. Should be :func:\`to_datetime\` (single backticks, no `pd`).